python/q617_testing_script.py

```diff
@@ -0,0 +1,124 @@
+#The following code imports the required packages.
+import math
+import json
+import os
+import pandas as pd
+import random
+import string
+import time
+import yaml
+import shutil as sh
+import utils
+
+from large_test_files import unique_uli
+
+"""
+This script creates 5 files for testing the Q617 edit with the same
+loan amount, property value, and CLTV. Each file has CLTV rounded 
+to a different value. Each file has loan amounts and property values 
+rounded to either 0, 1, or two decimal places. 
+
+CLTV is calculated by dividing loan amount by the property value for each row
+and rounding to the place specified in the filename. 
+
+"""
+
+def passes_cltv(cltv_rounded = 0, row=None):
+    """Creates CLTV values from loan amount
+    and property value"""
+    
+    #Divides loan amount by provery value.
+    ltv = row['loan_amount']/row['property_value']
+    
+    #Calculates CLTV by rounding LTV to a place specified in the function.
+    cltv = round(ltv, cltv_rounded)
```

Comment:
Is cltv_rounded a variable holding number of decimal places?

If so, can we change the name to indicate its function?